backend/signing/keypath.go

```diff
@@ -32,6 +32,9 @@ const (
 
 	// NonHardened denotes a non-hardened key derivation.
 	NonHardened = false
+
+	// HARDENED is the BIP44 offset to make a keypath element hardened.
+	HARDENED uint32 = hdkeychain.HardenedKeyStart
```

Comment:
Why not use directly `hdkeychain.HardenedKeyStart`? I'd find it very confusing to see `signing.Hardened` and `signing.HARDENED`  - they look too similar.